.github/workflows/angular.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - run: npm install
       - run: npm run build --if-present
       - run: npm run test -- --watch=false --browsers=ChromeHeadless
+      - run: npm run pre:e2e
```

Comment:
Implemented as a temporary workaround due to version mismatch of chromedriver in E2E test.